metadata-transformer/src/main/java/no/unit/nva/metadata/DcTerms.java

```diff
@@ -80,6 +80,10 @@ public IRI getIri(ValueFactory valueFactory) {
         return valueFactory.createIRI(DCTERMS_PREFIX, localName);
     }
 
+    public String getDcLocalName() {
```

Comment:
is localName the official term for... whatever comes after the dot? If not, maybe getTerm() would be more descriptive? Just a thought.

Reply:
localName here is the URI local name for a property, either the last element in the path or the string in fragment. These should be accessed only via the IRI::getLocalName, I think.